src/Sylius/Behat/CountryNameToCodeConverterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Sylius\Behat;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface CountryNameToCodeConverterInterface
+{
+    /**
+     * @param string $name
+     *
+     * @return string
+     *
+     * @throws \InvalidArgumentException If name is not found in country code registry.
```

Comment:
It knows nothing, but we can declare, that implementations will throw given type of exception if something bad happens, so the interface consumer can catch it and handle it. Throwing an exception is one of the ways to communicate between objects just like returning a value.